### PR TITLE
Feature/ge author page

### PR DESCRIPTION
**Pull request by GreysonElkins**
#### What's this PR do?
* Get's an author page set up where users can edit their profiles
#### Where should the reviewer start?
* UserPage component
#### How should this be manually tested?
* Make a user! Give them a profile!

### CODE REVIEW

**Review by nickhartdev — APPROVED** [review]
Super cool feature! Looks good!

**Review by carlymclift — APPROVED** [review]
Really cool feature to add! awesome job!

File: src/components/App/App.js [hunk 1]
```diff
@@ -16,7 +17,14 @@ class App extends Component {
...
       error: '',
-      currentUser: {},
+      currentUser: {
+        id: null,
+        name: '',
+        email: '',
+        bio: null,
+        created_at: '',
+        updated_at: ''
+      },
```
Comment (Abdeboskey) [inline src/components/App/App.js:27]: Very nice packaging here

**Review by Abdeboskey — APPROVED** [review]
This is great, and opens up lots of doors for future iterations!!

**Pull request merged**